packages/ws-client/src/data-source.ts

```diff
@@ -22,14 +28,21 @@ export type UpdateWebsocketMessage = {
   val: JSONValue
 }
 
+export type EventResponseWebsocketMessage = {
+  $: 'evt-res'
+  key: string
+  val: JSONValue
```

Comment:
Lets maybe have one value for resolved and a different value for rejected/caught

Reply:
Could be, tricky to parse otherwise!